src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/WebViewPage.java

```diff
@@ -0,0 +1,29 @@
+package com.qaprosoft.carina.demo.mobile.gui.pages.android;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.demo.mobile.gui.pages.common.ContactUsPageBase;
+import com.qaprosoft.carina.demo.mobile.gui.pages.common.WebViewPageBase;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = WebViewPageBase.class)
+public class WebViewPage extends WebViewPageBase {
+
+
+    @FindBy(className = "t708__btn")
+    private ExtendedWebElement contactUsLink;
+
+    public WebViewPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public ContactUsPageBase goToContactUsPage() {
+        changeToWebViewContext(getDriver());
```

Comment:
- Please remove changeToWebViewContext from method and add if you need into test